dependencies/CMakeLists.txt

```diff
@@ -147,7 +147,7 @@ AddDependency(NAME       BTK
 
 AddDependency(NAME       simbody
               URL        https://github.com/simbody/simbody.git
-              TAG        9f2123212dadcedcd680a8d8b3832d4fe40e1f8b
+              TAG        247f0cdb2916317115e50186c2f3ee0a177e02b3
```

Comment:
This commit of Simbody includes the feature where the search path for simbody-visualizer includes the relative path from libSimTKsimbody. This PR is the first to copy over simbody-visualizer into OpenSim on UNIX, and so including this commit allowed testing that the simbody-visualizer is indeed found properly (I tested this on mac).